plugins/postgres/dbt/adapters/postgres/connections.py

```diff
@@ -147,5 +148,13 @@ def get_credentials(cls, credentials):
         return credentials
 
     @classmethod
-    def get_status(cls, cursor):
-        return cursor.statusmessage
+    def get_status(cls, cursor) -> ExecutionStatus:
+        message = str(cursor.statusmessage)
+        rows = cursor.rowcount
```

Comment:
I don't think we should intervene, _well_ actually I don't feel comfortable intervening since I don't know all the possibilities that could be returned by cursor API.